doc/devdocs/meta.rst

```diff
@@ -46,7 +46,7 @@ necessary. If ``args`` is specified, a nested expression containing
 to specify additional information.
 
 To use the metadata, you have to parse these ``:meta`` expressions.
-If your implementation can be performed within Julia, ``popmeta!`` is
+If your implementation can be performed within Julia, ``Base.popmeta!`` is
 very handy: ``popmeta!(body, :symbol)`` will scan a function *body*
```

Comment:
I was trying not only replace them in the first definition location so that the file wasn't overloaded with `Base.`, but if you think it would be best to change that one too. 

Done.